Engineering-Reference/M365SourceAO.md

```diff
@@ -182,15 +169,13 @@ Default Value: SMTP
 
 Choose either SMTP or SIP address when resolving contact email addresses.
 
-Change Conditions: ??
-
 ### Exclude Group GAL Contact Aliases <a name="exludegalcontact"></a>
 {: .no_toc }
-Default Value: SMTP
+Default Value: Off
 
 Exclude additional email addresses (aliases) when resolving a Group GAL contact.
 
-Change Conditions: ??
+Change Conditions: Turn on to exclude alias addresses in GAL contacts.
```

Comment:
?? think this is turn on if you want the aliases inside a group resolved or off if you want to just use the group alias.

Reply:
done